user-guide/Advanced_Functionality/Security/Basic_security_configuration/Managing_user_groups/Configuring_a_user_group.md

```diff
@@ -35,27 +35,54 @@ To configure a user group:
+1. To configure notifications for the group, go to the *Alerts* tab and follow the same procedure as for alerts for a single user. See [Configuring user notifications](xref:Configuring_user_notifications).
 
-   - **Power users**: Provides full access to the DMS, without any permission to change the DMS configuration.
+   > [!NOTE]
+   > In contrast to alerts for a single user, alerts for a group cannot be sent with Alerter.
 
-   - **Administrators (read-only access to security)**: Selects all rights, except security configuration rights.
+1. Click *Apply*.
 
-   - **Administrators**: Selects all rights.
+## Using presets to assign rights
 
-   > [!TIP]
-   > See also:
-   >
-   > - [DataMiner user permissions](xref:DataMiner_user_permissions)
-   > - [Which user permissions are included in which security preset?](xref:Frequently_asked_questions_about_user_group_settings#which-user-permissions-are-included-in-which-security-preset)
+When you are configuring the rights for a group in the *Users / Groups* section in System Center, instead of selecting the different separate rights, you can select one of the available presets. From DataMiner 10.4.0 [CU12]/10.5.0 [CU1]/10.5.3 onwards<!-- RN 41656 -->, the presets are available in a separate section on the right. In earlier DataMiner versions, you can select them in a dropdown box at the top of the card.
 
-1. To configure notifications for the group, go to the *Alerts* tab and follow the same procedure as for alerts for a single user. See [Configuring user notifications](xref:Configuring_user_notifications).
+The following presets are available by default:
 
-   > [!NOTE]
-   > In contrast to alerts for a single user, alerts for a group cannot be sent with Alerter.
+- **No rights**: Deselects all rights.
 
-1. Click *Apply*.
+- **Operators**: Provides access to basic DataMiner functionality, without any permission to edit or delete elements, services, Automation scripts, documents, etc.
+
+- **Power users**: Provides full access to the DMS, without any permission to change the DMS configuration.
+
+- **Administrators (read-only access to security)**: Selects all rights, except security configuration rights.
+
+- **Administrators**: Selects all rights.
+
+In addition to these default presets, from DataMiner 10.4.0 [CU12]/10.5.0 [CU1]/10.5.3 onwards, custom presets are also supported. You can manage these with the buttons at the top of the pane on the right:
+
+| Button | Description |
+|--|--|
+| ![Reset button](~/user-guide/images/Security_preset_reset.png) | Reverts all changes made to the rights of the currently selected user group. |
+| ![Export button](~/user-guide/images/Security_preset_export.png) | Exports the currently selected rights/preset to a preset file (in JSON format). |
+| ![Import button](~/user-guide/images/Security_preset_import.png) | Imports a preset file. |
```

Comment:
When you import a preset file, does it only get added to the list, or is it also applied to the selected group?